README.md

```diff
@@ -212,6 +212,7 @@ A curated list of awesome Java frameworks, libraries and software.
 * [Jenkins](http://jenkins-ci.org/) - Provides server-based deployment services.
 * [TeamCity](http://www.jetbrains.com/teamcity/) - JetBrain's CI solution with a free version.
 * [Travis](https://travis-ci.org) - Hosted service often used for open-source projects.
+* [Concourse](https://concourse.ci/) - Provides dockerized pipeline with configuration that can be save in version control.
```

Comment:
Doesn't Travis have a thing where you run CI jobs inside containers?

I think the unique thing about concourse is it's fully open source and it's not really a platform per sae but a tool which can be integrated into other systems,. AFAIK no other CI platform is like this. But I may be wrong about all this, as I just quickly went over the website and I'm not familiar with how the other CI platforms are developed and managed.